lib/validators/webLedgerConfiguration.js

```diff
@@ -70,29 +68,19 @@ module.exports = async ({ledgerNode, validatorInput}) => {
 };
 
 function _getPublicKey(publicKeyId) {
-  const publicKeyBase58 = _getPublicKeyFromId(publicKeyId);
+  const publicKeyMultibase = _getPublicKeyFromId(publicKeyId);
   return {
     id: publicKeyId,
-    type: 'Ed25519VerificationKey2018',
+    type: 'Ed25519VerificationKey2020',
     controller: publicKeyId,
-    publicKey: publicKeyId,
-    publicKeyBase58
+    publicKeyMultibase
   };
 }
 
 function _getPublicKeyFromId(publicKeyId) {
   const parsed = URL.parse(publicKeyId);
   // drops the # from the front
   const mbPubkey = parsed.hash.substr(1);
-  const mcPubkeyBytes = multibase.decode(mbPubkey);
-  const mcType = multicodec.getCodec(mcPubkeyBytes);
-  if(mcType !== 'ed25519-pub') {
-    throw new BedrockError(
-      'Key ID is not a multiformats encoded ed25519 public key.',
-      'EncodingError', {publicKeyId});
-  }
```

Comment:
Why is it ok to remove this validation?

Reply:
... because the only place this is called from the code is just before it passes it to `Ed25519VerificationKey2020.from(computedKey)`... which re-does that exact same check in that code. Key format checking should be deferred to the key implementations... we shouldn't duplicate key format checking code here.